superset/db_engine_specs/druid.py

```diff
@@ -100,3 +100,17 @@ def convert_dttm(cls, target_type: str, dttm: datetime) -> Optional[str]:
         if tt in (utils.TemporalType.DATETIME, utils.TemporalType.TIMESTAMP):
             return f"""TIME_PARSE('{dttm.isoformat(timespec="seconds")}')"""
         return None
+
+    @classmethod
+    def epoch_to_dttm(cls) -> str:
+        """
+        Convert from number of seconds since the epoch to a timestamp.
+        """
+        return "MILLIS_TO_TIMESTAMP({col} * 1000)"
+
+    @classmethod
+    def epoch_ms_to_dttm(cls) -> str:
+        """
+        Convert from number of milliseconds since the epoch to a timestamp.
+        """
+        return "MILLIS_TO_TIMESTAMP({col})"
```

Comment:
Oh wow, I didn't know these weren't defined yet for Druid!

Reply:
Yeah, I implemented `epoch_to_dttm` while working on a solution, and decided to leave it there even if I wasn't using it, and add `epoch_ms_to_dttm` for completeness.